scripts/releaseTag.mjs

```diff
@@ -67,7 +67,7 @@ async function main(argv) {
     );
   }
 
-  await exec(['git', 'push', muiXRemote.name, tag].join(' '));
+  await exec(['git', 'push', muiXRemote.name, `refs/tags/${tag}`].join(' '));
```

Comment:
WDYT about going with a shorthand instead? 🤔 
https://stackoverflow.com/a/49906565/3303436
IMHO, it's arguably even more readable. 🙈 
```suggestion
  await exec(['git', 'push', muiXRemote.name, 'tag', tag].join(' '));
```